src/runtime/hexagon_remote/qurt/sim_remote.cpp

```diff
@@ -152,12 +152,17 @@ int release_library(handle_t module_ptr) {
 }
 
 extern "C" {
-halide_profiler_instance_state profiler_state;
-int *profiler_current_func_addr = &profiler_state.current_func;
+halide_profiler_instance_state hvx_profiler_instance;
+int *profiler_current_func_addr = &hvx_profiler_instance.current_func;
+
+halide_profiler_instance_state *halide_hexagon_remote_profiler_get_global_instance() {
```

Comment:
Did you on purpose create two separate instances of `halide_profiler_state`?

Reply:
Thanks for noticing. Fixed it.